test/cbmc/proofs/OtaSendEvent_FreeRTOS/OtaSendEvent_FreeRTOS_harness.c

```diff
@@ -0,0 +1,42 @@
+/*
+ * AWS IoT Over-the-air Update v3.1.0
+ * Copyright (C) 2021 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+/**
+ * @file OtaSendEvent_FreeRTOS_harness.c
+ * @brief Implements the proof harness for OtaSendEvent_FreeRTOS function.
+ */
+/*  FreeRTOS includes for OTA library. */
+#include "ota_os_freertos.h"
+
+void OtaSendEvent_FreeRTOS_harness()
+{
+    OtaEventContext_t * pEventCtx;
+    OtaOsStatus_t osStatus;
+
+    void * pEventMsg;
+    unsigned int timeout;
+
+    osStatus = OtaSendEvent_FreeRTOS( pEventCtx, pEventMsg, timeout );
```

Comment:
I see that the function body for `xQueueSendToBack` has been removed. I just want to make sure that we have confirmed that `pEventMsg` is non-`NULL` when OtaSendEvent_FreeRTOS called from OTA library as `xQueueSendToBack` has an assert on it.

Reply:
Have we checked that?

Reply:
Added the assumption for pEventMsg in d4c6031